wasmtime-jni/src/io_github_kawamuray_wasmtime_Linker/imp.rs

```diff
@@ -66,6 +67,48 @@ impl<'a> JniLinker<'a> for JniLinkerImpl {
         Ok(ret)
     }
 
+    fn native_externs(
+        env: &JNIEnv,
+        this: JObject,
+        store_ptr: jlong,
+        module: JString,
+    ) -> Result<jobjectArray, Self::Error> {
+        let mut store = interop::ref_from_raw::<Store<StoreData>>(store_ptr)?;
+        let linker = interop::get_inner::<Linker<StoreData>>(env, this)?;
+        let module = utils::get_string(env, *module)?;
+        let mut vec: Vec<String> = Vec::new();
+        for item in linker.iter(&mut *store) {
+            if item.0.eq(&module) {
```

Comment:
Let's use `==` operator instead following idiom.

Reply:
I guess. Seems to work.